flybywiresim/a32nx
# fix(efb): fix ground services page being blank

Fixes #7412

## Summary of Changes



Disabling the pushback map's center map on plane option while in flight used to cause the screen elements to stretch and eventually glitch out the EFB's ground services page. This pull request fixes the issue.

https://cdn.discordapp.com/attachments/747571985831297145/1010776841012908173/FlightSimulator_znnW2lpFfE.webm


## Screenshots (if necessary)




## References

N/A




## Additional context

N/A


Discord username (if different from GitHub): 2Cas#1022

## Testing instructions

1. Start a flight with the EFB open to the pushback page
2. Disable the center map on plane option 

![image](https://user-images.githubusercontent.com/15316958/185780564-d4691fcc-7ff1-46b9-845d-7824251eeb55.png)
3. Fly in a straight line for 2 minutes.
4. The ground services page should still work.
5. Reenable the center map on plane option, this should also still function as expected.
6. Repeat, but toggle between the exterior and interior camera, as well as different EFB pages.
7. Open the pushback page, disable the center map on plane option, open the EFB to a different page
8. Either continue to your destination airfield or return to the origin airfield, land, and open the pushback page again once on ground. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
